src/components/LayerCollection/LayerPOLine.js

```diff
@@ -156,18 +166,40 @@ class LayerPOLine extends Component {
     }
   };
 
-  updatePOLine = (data) => {
+  openOrder = async (saveAndOpen) => {
+    const { mutator } = this.props;
+    const order = this.getOrder();
+    const updatedOrder = await mutator.order.PUT({
```

Comment:
are you sure about this? it sends request to update order in both cases

Reply:
Fixed